docs/installation-and-operations/configuration/environment/README.md

```diff
@@ -61,79 +61,109 @@ Please note: The Configuration is a HashWithIndifferentAccess and thus it should
+OPENPROJECT_ENTERPRISE__PLAN: "enterprise-on-premises---euro---1-year"
+OPENPROJECT_FOG__DOWNLOAD__URL__EXPIRES__IN: 21600
+OPENPROJECT_FORCE__HELP__LINK: nil
+OPENPROJECT_FORCE__FORMATTING__HELP__LINK: nil
+OPENPROJECT_HEALTH__CHECKS__AUTHENTICATION__PASSWORD: nil
+OPENPROJECT_HEALTH__CHECKS__JOBS__QUEUE__COUNT__THRESHOLD: 50
+OPENPROJECT_HEALTH__CHECKS__JOBS__NEVER__RAN__MINUTES__AGO: 5
+OPENPROJECT_HEALTH__CHECKS__BACKLOG__THRESHOLD: 20
+OPENPROJECT_GRAVATAR__FALLBACK__IMAGE: "404"
+OPENPROJECT_HIDDEN__MENU__ITEMS: {}
+OPENPROJECT_IMPRESSUM__LINK: nil
+OPENPROJECT_INSTALLATION__TYPE: "docker"
+OPENPROJECT_INTERNAL__PASSWORD__CONFIRMATION: true
+OPENPROJECT_LDAP__FORCE__NO__PAGE: nil
+OPENPROJECT_LDAP__AUTH__SOURCE__TLS__OPTIONS: nil
+OPENPROJECT_LDAP__GROUPS__DISABLE__SYNC__JOB: false
+OPENPROJECT_LDAP__TLS__OPTIONS: {}
+OPENPROJECT_LOG__LEVEL: "info"
+OPENPROJECT_LOGRAGE__FORMATTER: nil
+OPENPROJECT_MAIN__CONTENT__LANGUAGE: "english"
+OPENPROJECT_MIGRATION__CHECK__ON__EXCEPTIONS: true
+OPENPROJECT_OMNIAUTH__DIRECT__LOGIN__PROVIDER: nil
+OPENPROJECT_OVERRIDE__BCRYPT__COST__FACTOR: nil
+OPENPROJECT_ONBOARDING__VIDEO__URL: "https://player.vimeo.com/video/163426858?autoplay=1"
+OPENPROJECT_ONBOARDING__ENABLED: true
+OPENPROJECT_RAILS__ASSET__HOST: nil
+OPENPROJECT_RAILS__CACHE__STORE: :file_store
+OPENPROJECT_RAILS__RELATIVE__URL__ROOT: ""
+OPENPROJECT_RAILS__FORCE__SSL: false
+OPENPROJECT_REGISTRATION__FOOTER: {"en"=>""}
+OPENPROJECT_SCM: {}
+OPENPROJECT_SCM__GIT__COMMAND: nil
+OPENPROJECT_SCM__LOCAL__CHECKOUT__PATH: "repositories"
+OPENPROJECT_SCM__SUBVERSION__COMMAND: nil
+OPENPROJECT_SECURITY__BADGE__DISPLAYED: true
+OPENPROJECT_SECURITY__BADGE__URL: "https://releases.openproject.com/v1/check.svg"
+OPENPROJECT_SENDMAIL__ARGUMENTS: "-i"
+OPENPROJECT_SENDMAIL__LOCATION: "/usr/sbin/sendmail"
+OPENPROJECT_SENTRY__BREADCRUMB__LOGGERS: ["active_support_logger"]
+OPENPROJECT_SENTRY__DSN: nil
+OPENPROJECT_SENTRY__FRONTEND__DSN: nil
+OPENPROJECT_SENTRY__HOST: nil
+OPENPROJECT_SENTRY__TRACE__FACTOR: 0
+OPENPROJECT_SENTRY__FRONTEND__TRACE__FACTOR: 0
+OPENPROJECT_SESSION__COOKIE__NAME: "_open_project_session"
+OPENPROJECT_SESSION__STORE: :active_record_store
+OPENPROJECT_SHOW__COMMUNITY__LINKS: true
+OPENPROJECT_SHOW__PENDING__MIGRATIONS__WARNING: true
+OPENPROJECT_SHOW__SETTING__MISMATCH__WARNING: true
+OPENPROJECT_SHOW__STORAGE__INFORMATION: true
+OPENPROJECT_SHOW__WARNING__BARS: true
+OPENPROJECT_SMTP__ADDRESS: ""
+OPENPROJECT_SMTP__AUTHENTICATION: "plain"
+OPENPROJECT_SMTP__DOMAIN: "your.domain.com"
+OPENPROJECT_SMTP__ENABLE__STARTTLS__AUTO: false
+OPENPROJECT_SMTP__OPENSSL__VERIFY__MODE: "none"
+OPENPROJECT_SMTP__PASSWORD: ""
+OPENPROJECT_SMTP__PORT: 587
+OPENPROJECT_SMTP__USER__NAME: ""
+OPENPROJECT_SQL__SLOW__QUERY__THRESHOLD: 2000
+OPENPROJECT_STATSD: {"host"=>nil, "port"=>8125}
+OPENPROJECT_WEB: {"workers"=>2, "timeout"=>120, "wait_timeout"=>10, "min_threads"=>4, "max_threads"=>16}
```

Comment:
@adam-op Ah one more thing, could you please "expand" these nested values? I.e. don't give the hash as a default but list the individual values into separate variables. E.g. not just OPENPROJECT_WEB with the hash as the default but `OPENPROJECT_WEB_WORKERS: 2`, `OPENPROJECT_WEB_TIMEOUT: 120` etc..

Same with OPENPROJECT_STATSD.

Reply:
i will wait until the new rake task is implemented